packages/components/app/styles/components/dropdown.scss

```diff
@@ -113,33 +118,32 @@
   box-shadow: var(--token-surface-high-box-shadow);
   list-style: none;
   margin: 0;
-  max-width: 25rem;
-  min-width: 12.5rem;
+  max-width: 400px;
```

Comment:
[issue] [a11y] these need to be in relative units to support browser zoom. 

The most important reason for using responsive and unitless values in our CSS is for supporting our users that rely on zooming. If you read the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/), our users need to be able to zoom the viewport without loss of content or functionality, or restrictions imposed by CSS values or viewport scaling settings. 

In particular, there are a some success criteria to be met:

- [WCAG 1.4.4](https://www.w3.org/TR/WCAG21/#resize-text): Users must be able to resize text without assistive technology up to 200 percent, without loss of content or functionality. (Level AA)
- [WCAG 1.4.10](https://www.w3.org/TR/WCAG21/#reflow): Users must be able to resize text without being forced to scroll both horizontally and vertically to read that content. (Level AA)

Reply:
@MelSumner can you tell me how declaring the size in `px` or `rem` would change the outcome in this case (zooming, not changing base font size, which is something we're not able to support yet)? 

**This is the content at 200% with `px`**

<img width="1063" alt="px1" src="https://user-images.githubusercontent.com/686239/163986516-bdf48d8e-1de1-4c5d-a2d9-cbda5f422f34.png">
<img width="1056" alt="px2" src="https://user-images.githubusercontent.com/686239/163986510-3d9e8309-e8ba-4577-adfa-25d4398dedc3.png">

**This is the content at 200% with `rem`**

<img width="1065" alt="rem1" src="https://user-images.githubusercontent.com/686239/163986574-c1a3dae9-87fe-4a9d-865e-083a03f9b954.png">
<img width="1070" alt="rem2" src="https://user-images.githubusercontent.com/686239/163986581-bfe1d707-3df5-4a85-b664-61544b470c12.png">

They look exactly the same to me.